src/k2/transport/Status.h

```diff
@@ -450,6 +450,11 @@ static const inline Status S527_Railgun_Error{.code=527, .message=""};
 // Used by Qualys in the SSLLabs server testing API to signal that the site can't process the request.
 static const inline Status S529_Site_is_overloaded{.code=529, .message=""};
 
+// 530 Plog Id is duplicated 
+// Used by Plog Server to signal that the Plog Id is duplicated
+static const inline Status S530_PlogId_is_duplicated{.code=530, .message=""};
```

Comment:
You should not create custom codes. This file contains all HTTP standard status codes. We should be able to find one which suits . In particular, this error is not retryable so it should not be 5xx series code but rather 4xx series. I think 409_Conflict would work well here. A bit less descriptive but also usable code is 403_Forbidden

Reply:
fixed